services/dynamic-sidecar/tests/conftest.py

```diff
@@ -168,6 +169,7 @@ def mock_rabbit_check(mocker: MockerFixture) -> None:
 
 @pytest.fixture
 def base_mock_envs(
+    use_in_memory_long_running_tasks: None,
```

Comment:
again. is this a good idea to test with a different setup than what is running in prod?